package.json

```diff
@@ -72,7 +76,8 @@
       "unicorn/prefer-includes": "off"
     },
     "ignores": [
-      "strip-markdown.js"
+      "strip-markdown.js",
+      "types/index.d.ts"
```

Comment:
added this because `eslint-plugin-import` was complaining. alternatively maybe add `"unified": "*"` as a `peerDependency`?

Reply:
yes, it’s fine to ignore types in xo, it’s a bit annoying there